css/properties.schema.json

```diff
@@ -175,6 +176,8 @@
         "beforeAndAfterPseudos",
         "blockContainerElements",
         "blockContainers",
+        "blockContainersFlexContainersGridContainers",
```

Comment:
Nit: this should go after the two `blockContainersExcept*` lines, to keep alpha ordering.